### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by Ch-Haseeb** [thoughtco/ti-ext-maxorders#8]
### What happened?

I encountered an issue where I included a category in the timeslot_categories array, but I don't want it to affect the limit count. However, the code still reduces the timeslot based on that category.

### What did you expect to happen?

I would like to modify the behavior of the code so that when I include a category ID in the timeslot_categories array, orders belonging to that category are not counted towards the maximum order limit

### Version

1.1.1

**Comment by ryanmitchell** [comment]
This is a modification to the intention of the extension so would be something you would need to implement yourself.

**Comment by Ch-Haseeb** [comment]
Please tell me about the functionality of the related category records in the timeslot_categories table.

**Comment by ryanmitchell** [comment]
Im unclear as to what you are asking? the categories allow you to limit order limiting to items in those categories. It only applies when the restriction type is covers.

**Comment by Ch-Haseeb** [comment]
Yeah, but I removed the category ID from that table. However, it still limits the orders for all items in that category. How can I solve this issue? What should I check? I would appreciate it if you could provide some guidance.

**Comment by ryanmitchell** [comment]
Im really confused as to the issue as you are saying conflicting things. 
Can you please start over a provide a full description so I can have an understanding as to what you are seeing and expecting.

**Comment by Ch-Haseeb** [comment]
I have a category called 'cold drink,' and I want to exclude all items in that category from the order limiting functionality. Even though I removed the category ID from the 'time_slot_categories' table, the order limiting still applies to that category. What steps should I take to resolve this issue?

**Comment by ryanmitchell** [comment]
Im not able to replicate that - it's working fine on our installs. It would be a case of debugging the listener code yourself to see whats not working and raise a pull request for any changes you think are required.

**Comment by Ch-Haseeb** [comment]
Okay thanks
